extensions/wikia/Wall/WallHistoryController.class.php

```diff
@@ -1,5 +1,7 @@
 <?php
```

Comment:
Changes in this file related to [SOC-578](https://wikia-inc.atlassian.net/browse/SOC-578)